app/workers/evss/disability_compensation_form/job_status.rb

```diff
@@ -29,7 +29,7 @@ def job_exhausted(msg)
             error_message: error_message,
             updated_at: Time.now.utc
           }
-          Form526JobStatus.upsert({ job_id: jid }, values)
```

Comment:
the upsert gem and  [Activerecord.upsert ](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert)take different arguments